math-plugin/src/test/java/com/github/underscore/math/MathTest.java

```diff
@@ -279,84 +279,84 @@ public void chain() {
         $.chain(new String[] {""}).first(1);
         $.chain(new String[] {""}).firstOrNull();
         $.chain(new String[] {""}).firstOrNull(new Predicate<String>() {
-            public Boolean apply(String str) { return true; } });
+            public boolean test(String str) { return true; } });
         $.chain(new String[] {""}).initial();
         $.chain(new String[] {""}).initial(1);
         $.chain(new String[] {""}).last();
         $.chain(new String[] {""}).last(1);
         $.chain(new String[] {""}).lastOrNull();
         $.chain(new String[] {""}).lastOrNull(new Predicate<String>() {
-            public Boolean apply(String str) { return true; } });
+            public boolean test(String str) { return true; } });
         $.chain(new String[] {""}).rest();
         $.chain(new String[] {""}).rest(1);
         $.chain(new String[] {""}).compact();
         $.chain(new String[] {""}).compact("1");
         $.chain(new String[] {""}).flatten();
-        $.chain(new Integer[] {0}).map(new Function1<Integer, Integer>() {
+        $.chain(new Integer[] {0}).map(new Function<Integer, Integer>() {
```

Comment:
\[FindBugs\] INFO: RV: Return value of $$Chain\.map\(Function\) ignored, is this OK in com\.github\.underscore\.math\.MathTest\.chain\(\)